modules/apps/bean-portlet/bean-portlet-cdi-extension-sample/src/main/webapp/WEB-INF/liferay-plugin-package.properties

```diff
@@ -0,0 +1,11 @@
+author=Liferay, Inc.
+Bundle-Version: 1.0.0
+change-log=
+licenses=LGPL
+module-group-id=liferay
+module-incremental-version=1
+name=Sample Bean Portlet
+page-url=http://www.liferay.com
+tags=sample
+${osgi.dynamic.import.package}
+${osgi.import.package}
```

Comment:
@Kvale1733, please remove `${osgi.dynamic.import.package}` and `${osgi.import.package}`.